test/fixtures/go-binaries/symbols/source/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```

Comment:
A thought - do you think that we need to have this script as part of our `build` script, and remove the binary files from git?

Reply:
I don't think that makes a lot of sense as this takes a "long" time to build (~5m). Also, the binaries should be reproducible, so there's no real motivation to rebuild them again. I've kept this more as a reference for future builds (with newer Go versions) and that we know what is in these binaries :-)

Reply:
Alright, I'm fine with that.

Reply:
Yeah me neither, but I think it's the easiest & fastest thing to do... :/